sdk: Generate php sdk and point php autoloader to lib folder

pnicolcev-tulipretail opened the pull request:
Generate php sdk and include it in the project.
Point php autoloader to lib folder.

arekkas commented:
That look pretty solid already, thank you!

Could you add a quick guide on how to instantiate this to [docs/sdk/php.md](https://github.com/ory/hydra/tree/master/docs/sdk)?

arekkas commented:
ping @pnicolcev-tulipretail :)

pnicolcev-tulipretail commented:
@arekkas I'm here! Added and squashed. You may also want to fork/take ownership of the oauth2 client: https://github.com/pnicolcev-tulipretail/oauth2-hydra which is referenced in the php.md.

Also please register on packagist.org and submit this sdk once you merge so that composer users have an easier time pulling in your sdk.

arekkas commented on docs/sdk/php.md:
````diff
@@ -0,0 +1,90 @@
+```json
+{
+    "repositories": {
+        "tulip/hydra-php-sdk": { "type": "vcs", "url": "git@git.internal.tulip.io:shared/composer-libraries/hydra-php-sdk.git" }
````
I guess this can be updated? I registered packagist here: https://packagist.org/packages/ory/hydra-sdk

pnicolcev-tulipretail replied:
Oops, the above was from internal testing. Fixed this reference to your packagist repo.

arekkas commented on docs/sdk/php.md:
````diff
@@ -0,0 +1,90 @@
+client that has had this work done for us.
+
+```sh
+composer require tulip/oauth2-hydra
````
Same thing here I guess?

pnicolcev-tulipretail replied:
Different repo. I can change the org to ory but maybe it's better if you take ownership if it's official. You can fork this guy https://github.com/tulipretail/oauth2-hydra, change tulip/oauth2-hydra to ory/oauth2-hydra in `composer.json` and submit it to packagist.

You could also make it official by making a PR to add it to the league's list https://github.com/thephpleague/oauth2-client/blob/master/docs/providers/thirdparty.md

arekkas commented:
Thank you!

arekkas commented:
This PR was made against master which is currently scheduled for the first alpha of 1.0.0, but that will take a some time before being published.

Could you re-open this PR against the `0.11` branch? Then I can create a release for that version which includes the PHP SDK.

arekkas commented:
Damn, I see that the branch was deleted... Maybe you have a backup locally?

pnicolcev-tulipretail commented:
Hm, I just restored it. Will try opening one to 0.11
